portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/gogoshell/GogoShell.testcase

```diff
@@ -47,6 +47,19 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-189944. Verify no errors are thrown when running the disconnect command from Gogo Shell."
+	@priority = 3
+	test CanRunDisconnectCommandViaTelnet {
+		property test.assert.warning.exceptions = "true";
+		property test.liferay.virtual.instance = "false";
```

Comment:
@Kvale1733 Although this may seem a bit weird to spawn a virtual instance for this, it would technically be faster than using a solo bundle and then having to restart it for the next testcase. Since starting and disconnecting has basically 0 risk on affecting any other testcase, we should use the fastest setup available. Setting this to true instead